src/cmd/src/datanode.rs

```diff
@@ -236,6 +236,7 @@ mod tests {
             checkpoint_margin = 9
             gc_duration = '7s'
             checkpoint_on_startup = true
+            use_compress = true
```

Comment:
nitpick: What about `compress`?
```suggestion
            compress = true
```

Reply:
done